### DISCUSSION TITLE
Mobile support

### DISCUSSION THREAD

**Discussion by matheusfm** [monuelo/i-just-want-to-play-tetris#17] [Ideas]
First of all, thanks for this nice project!

I would like to play this game on my smartphone.
What do you think about this?

Now, it isn't possible because the game is played with keyboard.
Maybe adding some buttons in mobile version could help.

**Comment by 0xHericles** [comment]
@matheusfm I thank you for the suggestion. Besides button, I think we could add some gesture controls to actions like hard drop and moving.

**Comment by 0xHericles** [comment]
That's my suggestion for control mapping on mobile:

- **Hold** - Tap on the hold square
- **Rotate** - Tap on screen
- **Move (Left/Right)** - Swipe Left/Right 
- **Hard drop** - Swipe Down
- **Speed up drop** - Slightly swipe down
